tools/trace_analyzer_tool.h

```diff
@@ -213,6 +254,7 @@ class TraceAnalyzer {
   uint64_t total_access_keys_;
   uint64_t total_gets_;
   uint64_t total_writes_;
+  // To do: why no counters for seeks and multigets?
```

Comment:
Good catch! I forgot to add them........

Reply:
Added